src/auth0/handlers/actions.js

```diff
@@ -82,8 +109,7 @@ export const schema = {
               }
             }
           }
-        },
-        required: [ 'code', 'dependencies', 'runtime' ]
+        }
       },
       bindings: {
```

Comment:
Since we'll have bindings represented at the trigger level, I think we can remove this entirely from each action.

Reply:
yes, you are absolutely right!